storage/mysql/mysql.go

```diff
@@ -55,10 +60,25 @@ func New(ctx context.Context, db *sql.DB) (*Storage, error) {
 		}
 
 		klog.Infof("Initializing checkpoint")
-		if err := s.writeCheckpoint(ctx, []byte("")); err != nil {
+		// Get a Tx for making transaction requests.
+		tx, err := s.db.BeginTx(ctx, nil)
+		if err != nil {
+			return nil, err
+		}
+		// Defer a rollback in case anything fails.
+		defer func() {
+			if err := tx.Rollback(); err != nil && err != sql.ErrTxDone {
+				klog.Errorf("Failed to rollback in write initial checkpoint: %v", err)
+			}
+		}()
+		if err := s.writeCheckpoint(ctx, tx, 0, []byte("")); err != nil {
```

Comment:
Is `[]byte("")` the right value? That's the empty byte string, where it should be a _hash_ of the empty bytes. See https://github.com/transparency-dev/trillian-tessera/blob/main/cmd/example-posix/main.go#L102C24-L102C57 where it uses `rfc6962.DefaultHasher.EmptyRoot()`

Reply:
Updated to `rfc6962.DefaultHasher.EmptyRoot()`, thanks!